src/Orleans.SDK.targets

```diff
@@ -30,15 +30,14 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     </When>
     <Otherwise>
       <PropertyGroup>
-        <CodeGenToolExe>$(CodeGenToolExeDir)\ClientGenerator.exe</CodeGenToolExe>
+        <CodeGenToolExe>$(CodeGenToolExeDir)/ClientGenerator.exe</CodeGenToolExe>
       </PropertyGroup>
     </Otherwise>
   </Choose>
 
   <!-- This target is run just before Compile for an Orleans Grain Interface Project -->
   <Target Name="OrleansCodeGeneration"
-          AfterTargets="BeforeCompile"
-          BeforeTargets="CoreCompile"
+          BeforeTargets="AssignTargetPaths"
```

Comment:
Yes, this is part of the normal flow. @ReubenBond suggested this change, because generated files were not picked up in the earlier configuration by NuGet pack, since ```Compile``` item collection happened earlier.